docs/source/array-creation.rst

```diff
@@ -132,3 +148,73 @@ As an example, we might load a grid of pickle files known to contain 1000 by
     chunks = ((1000, 1000), (1000, 1000, 1000))
 
     x = da.Array(dsk, 'x', chunks)
+
+
+Store Dask Arrays
+=================
+
+In Memory
+---------
+
+If you have a small amount of data, you can call ``np.array`` on your dask array to tur
+in to a normal NumPy array:
+
+.. code-block:: Python
+
+   >>> x = da.arange(6, chunks=3)
+   >>> y = x**2
+   >>> np.array(y)
+   array([0, 1, 4, 9, 16, 25])
+
+
+HDF5
+----
+
+Use the ``to_hdf5`` function to store data into HDF5 using ``h5py``:
+
+.. code-block:: Python
+
+   >>> da.to_hdf5('myfile.hdf5', '/y', y)  # doctest: +SKIP
+
+Store several arrays in one computation with the function
+``da.to_hdf5`` by passing in a dict:
+
+.. code-block:: Python
+
+   >>> da.to_hdf5('myfile.hdf5', {'/x': x, '/y': y})  # doctest: +SKIP
+
+Other On-Disk Storage
+---------------------
+
+Alternatively, you can store dask arrays in any object that supports numpy-style
+slice assignment like ``h5py.Dataset``, or ``bcolz.carray``:
+
+.. code-block:: Python
+
+   >>> import bcolz  # doctest: +SKIP
+   >>> out = bcolz.zeros(shape=y.shape, rootdir='myfile.bcolz')  # doctest: +SKIP
+   >>> da.store(y, out)  # doctest: +SKIP
+
+You can store several arrays in one computation by passing lists of sources and
+destinations:
+
+   >>> da.store([array1, array2], [output1, outpu2])  # doctest: +SKIP
+
+
+On-Disk Storage
+---------------
+
+In the example above we used ``h5py``, but ``dask.array`` works equally well
+with ``pytables``, ``bcolz``, or any library that provides an array object from
+which we can slice out numpy arrays:
+
+.. code-block:: Python
+
+   >>> x = dataset[1000:2000, :2000]  # pull out numpy array from on-disk object
+
+This API has become a standard in Scientific Python.  Dask works with any
```

Comment:
Maybe use something other than `Scientific Python` to avoid confusion with SciPy, or the [ScientificPython](http://dirac.cnrs-orleans.fr/plone/software/scientificpython/) library?